CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.0.3]
+
```

Comment:
this needs to have links adjusted at the end of the file to work. please adjust or remove the version tag here

```suggestion
```